.travis.yml

```diff
@@ -24,6 +24,7 @@ env:
   - PACKAGE=libxml2
   - PACKAGE=libusb
   - PACKAGE=symbiflow-yosys
+  - PACKAGE=yosys-sv
```

Comment:
The convention is `<forkowner>-<package>` see bit.ly/edda-conda-eda-spec 

I think this should be `PACKAGE=antmicro-yosys`?

Reply:
Changed to `antmicro-yosys`